test/basic.html

```diff
@@ -29,43 +31,48 @@
   </test-fixture>
 
   <script>
-    suite('basic', function() {
-      var form;
+    window.addEventListener('WebComponentsReady', function() {
```

Comment:
oh right! this was needed before since we were skipping the whole test suite instantiation (we had to wait for Polymer to be defined)
This is not needed anymore, removing it now!